### DISCUSSION TITLE
COD WAW graphics

### DISCUSSION THREAD

**Discussion by TheRebreathing** [nzp-team/nzportable#492] [Other Content Releases]
![graphics](https://github.com/nzp-team/nzportable/assets/152328160/6461ca5f-bdb3-4278-92ac-03bfebc9b4fc)
Includes hud and main menu
[Download (PC, Switch, Vita)](https://www.mediafire.com/file/7kmcmtgqd7y5efm/cod+waw+graphics+for+nzp.zip/file)
[Download (3DS, PSP,)](https://www.mediafire.com/file/eg8ir5juzg7djox/cod+waw+graphics+for+nzp+psp.zip/file)

**Comment by Renatochon** [comment]
And how do I put it on android psp?

**Comment by 26lake** [comment]
Can you do CW graphics next?
